step-release-vis/src/app/components/environment/environment.ts

```diff
@@ -1,6 +1,7 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import {EnvironmentService} from '../../services/environment';
 import {Polygon} from '../../models/Polygon';
+import {random} from 'lodash';
```

Comment:
you could change this to "import random from 'lodash/random" if I am correct, so you do not load the whole lodash lib, but only a small part.

Reply:
This way TSLint gives a warning.

![image](https://user-images.githubusercontent.com/35143083/90403870-41218780-e0aa-11ea-9f32-a1d43561de84.png)

Also Angular imports its stuff from @angular/core this way, for example `import {Component, OnInit} from '@angular/core';` was autogenerated.

Reply:
Ok, fair enough.